src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyles = sourceString
+    .split(';')
+    .filter((declaration) => declaration.trim())
+    .map((declaration) => declaration.split(':'))
+    .reduce((accumulator, [key, value]) => {
+      accumulator[key.trim()] = value.trim();
+
+      return accumulator;
+    }, {});
+
+  return cssStyles;
```

Comment:
[CODE STYLE] - The variable `cssStyles` could be named more descriptively to reflect its purpose, such as `styleObject` or `styleRules`. This will improve code readability.